app/Http/Controllers/Admin/AuthController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Auth;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Requests\Admin\LoginRequest;
+
+class AuthController extends AdminBaseController
+{
+
+    protected $auth;
```

Comment:
put comment for every property please !!! I hope this is the last time I talk about it.